src/modules/api.js

```diff
@@ -0,0 +1,54 @@
+const form = document.querySelector('.form');
+const refreshBtn = document.getElementById('refresh-btn');
+const table = document.getElementById('recent-list');
+const inputName = document.getElementById('name');
+const inputScore = document.getElementById('score');
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iNLPApgiyuuDUgI7yzuD/scores/';
+
+const uploadScore = async (user, score) => {
+  const response = await fetch(`${url}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ user, score }),
+  });
+  const data = await response.json();
+  return data;
+};
+
+const getScores = async () => {
+  const response = await fetch(`${url}`);
+  const data = await response.json();
+  return data;
+};
+
+const refreshScores = async () => {
+  const scores = await getScores();
+
+  table.innerHTML = '';
+
+  scores.result.forEach((gamer) => {
+    const li = document.createElement('li');
+    li.classList.add('recent-score');
+    li.innerHTML = `${gamer.user}: ${gamer.score}`;
+    table.appendChild(li);
+  });
```

Comment:
- [ ] **[OPTIONAL]** If you notice, this piece of code is in charge of displaying the scores in the DOM. What if you extract this logic into a separate function, called, for example, `displayScores` that receives the array of scores? Let's say for example, that in the future, you also want to show all existing scores when the page first loads, this function will come in handy to avoid repeating yourself, plus you will be separating concerns and writing cleaner code! 🤓 🔧